admin/class-acc_user_importer-admin.php

```diff
@@ -606,6 +610,38 @@ private function proccess_expiry () {
 
 		$this->log_dual("Active members=$num_active, inactive members=$num_inactive");
 
+		// If option is set and there were membership changes, send email notification
+		// There is no checking done to ensure the notification email addresses are valid.
+		if (!empty($options['accUM_notification_emails']) &&
```

Comment:
Wondering if we should check here if the email address is valid here already? Something like this could work:
`!empty($options['accUM_notification_emails']) && is_email($options['accUM_notification_emails']) ...`

Reply:
Would 'is_email' work for a string which is a comma-separated list of emails?  If not, I guess I can break the strings into an array of strings and validate each one, but I figured it was not worth it.

Reply:
Not doing it for now.